### DISCUSSION TITLE
gitsign hunks <leader>gh

### DISCUSSION THREAD

**Discussion by abhijeetsrivastava** [LazyVim/LazyVim#443] [Q&A] [answered]
How is this working? The key map set is https://github.com/LazyVim/LazyVim/blob/f0b548af2f602d4498f528cd1a4db8c1c0e5c048/lua/lazyvim/plugins/editor.lua#L251

so if I press `<leader>gh` it would allow me to press S/u/R for various actions and same with `[h` but that h in `gh` and `[h` is acting as a vim motion for me it seems and the menu isn't opening.

Please help!

**Comment by dpetka2001** [comment]
Sorry if it seems an obvious question but are you inside a repo when trying these keymaps?

**Reply by abhijeetsrivastava** [reply]
replied you below instead of this thread https://github.com/LazyVim/LazyVim/discussions/443#discussioncomment-5333359

**Comment by folke** [comment, marked as answer]
What do you mean for various actions?

`[h` and `]h` jump between hunks (git changes you did not commit yet)

The other mappings start with `<leader>gh`, so not `gh`

**Reply by abhijeetsrivastava** [reply]
Yea, I do `<leader>g` and then when I press h it does the h movement and `[h` `]h` doesn't work. 

I tried running the actions directly without keymap and it worked. I also, do not have any conflict with this kepmap 

Edit: In my original post I didn't escape the angle bracket and hence leader is missing. Updating

**Reply by folke** [reply]
It works perfectly fine for me....

**Reply by folke** [reply]
You should do `<leader>gh`, not `gh`

**Reply by folke** [reply]
What does `:verbose nmap ]h` say?

**Reply by abhijeetsrivastava** [reply]
`:verbose nmap ]h` gives 

`n  ]           * <Cmd>lua require("which-key").show("]", {mode = "n", auto = true})<CR>
        Last set from Lua`

**Reply by folke** [reply]
then that means that gitsigns is not loaded for you for some reason.
You of course need to do that verbose command on an open file. Did you do that? What does `:Lazy` show? Is gitsigns loaded?

**Reply by abhijeetsrivastava** [reply]
yea, did that from an open file.

Found the bug: I was in my older config and gitsign was getting overloaded. Thanks for the help! sorry for taking your time.

**Comment by abhijeetsrivastava** [comment]
Yes, I am on the file which I am editing/half way done. Should I not be there? my bad I thought these settings would help me reach the all the edited positions and is better than `g;` shortcut then.

Would it work in only git satus? I checked the gitsign functions directly without keymaps and they work

**Reply by folke** [reply]
of course it only works in a git repo. What do you mean? It's to navigate between git hunks

**Reply by abhijeetsrivastava** [reply]
was replying to [dpetka2001](https://github.com/dpetka2001)
